makefile.shade

```diff
@@ -1049,16 +1049,19 @@ functions @{
             return CXX;
         }
 
-        foreach(var dir in Environment.GetEnvironmentVariable("PATH").Split(Path.PathSeparator))
+        foreach(var dir in Environment.GetEnvironmentVariable("PATH").Split(new string[] { ";" }, StringSplitOptions.None))
```

Comment:
This change is wrong. The path separation on non windows oses is ':' thats why we use the Path separator. Can you give an example or where things went wrong.